pkg/sources/http/http.go

```diff
@@ -192,17 +197,53 @@ func New(vertexInstance *dfv1.VertexInstance, writers []isb.BufferWriter, fetchW
 			forwardOpts = append(forwardOpts, forward.WithReadBatchSize(int64(*x.ReadBatchSize)))
 		}
 	}
-	forwarder, err := forward.NewInterStepDataForward(vertexInstance.Vertex, h, destinations, forward.All, applier.Terminal, fetchWM, publishWM, forwardOpts...)
+
+	if vertexInstance.Vertex.SpecifyUDTransformer() {
```

Comment:
can't we move this logic to forwarder.go? It makes more sense to make it to the forwarder, else implementing each source can get complicated.

Reply:
Should this be in the `source.go`? Each source's `New...` function should accept an `applier` interface and `WhereTo`, and then they can be used to pass to the forwarder.

Reply:
Will address.